packages/runtime/test/integration/consensus.ts

```diff
@@ -0,0 +1,125 @@
+import { aliceAcc, bobAcc } from "@algo-builder/algob/test/mocks/account";
```

Comment:
Please update the filename

Reply:
what do you suggest?

Reply:
maybe transaction-array-tests or something related to feature added

Reply:
ok